src/components/notifications/SMSC.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+    <div class="mb-3">
+        <label for="smsc-login" class="form-label">{{ $t("API Username") }}</label>
+        <i18n-t tag="div" class="form-text" keypath="wayToGetClickSendSMSToken">
+            <a href="https://smsc.kz/" target="_blank">{{ $t("here") }}</a>
+        </i18n-t>
+        <input id="smsc-login" v-model="$parent.notification.smscLogin" type="text" class="form-control" required>
+        <label for="smsc-key" class="form-label">{{ $t("API Key") }}</label>
+        <HiddenInput id="smsc-key" v-model="$parent.notification.smscPassword" :required="true" autocomplete="new-password"></HiddenInput>
+    </div>
+    <div class="mb-3">
+        <div class="form-text">
+            {{ $t("checkPrice", ['СМСЦ']) }}
+            <a href="https://smsc.kz/tariffs/" target="_blank">https://smsc.kz/tariffs/</a>
+        </div>
+    </div>
+    <div class="mb-3">
+        <label for="smsc-to-number" class="form-label">{{ $t("Recipient Number") }}</label>
+        <input id="smsc-to-number" v-model="$parent.notification.smscToNumber" type="text" minlength="8" maxlength="14" class="form-control" required>
+    </div>
+    <div class="mb-3">
+        <label for="smsc-sender-name" class="form-label">{{ $t("From Name/Number") }}</label>
+        <input id="smsc-sender-name" v-model="$parent.notification.smscSenderName" type="text" minlength="3" maxlength="11" class="form-control">
+        <div class="form-text">{{ $t("Leave blank to use a shared sender number.") }}</div>
+    </div>
+    <div class="mb-3">
+        <label for="smsc-platform" class="form-label">{{ $t("smscTranslit") }}</label><span style="color: red;"><sup>*</sup></span>
+        <select id="smsc-platform" v-model="$parent.notification.smscTranslit" class="form-select">
+            <option value="0">{{ $t("Default") }}</option>
+            <option value="1">Translit</option>
+            <option value="2">MpaHc/Ium</option>
```

Comment:
Are these options Translatable?

Reply:
The documentation is in English. Top right flag icon.
[Official documentation](https://smsc.kz/api/http/#menu)
paragraph translit. That's what they're called.
A sign that the message you want to translate in translit.
0 (default) – do not translate into transliteration.
1 – translate to translit as " translate".
2 – translate into transliteration as " mpaHc/Ium".

Reply:
What does "translate into transliteration" mean?

Reply:
This does only partially answer my question:
-  is `MpaHc/Ium` translatable into english?
- What provider is this translated by?

Reply:
Option 0 (default)
Send SMS Text "Подписаться на новости"
Option 1 (translit)
In Cyrillic, for example, "Подписаться на новости" converts to Latin "Podpisat'sja na novosti" Send SMS. Example service https://translit.ru/
Option 2 (mpaHc/Ium)
I did not use this option.


This is done to reduce the cost of SMS. Since SMS is more expensive in Cyrillic